Makefile

```diff
@@ -116,14 +92,14 @@ endif
 PACKAGE_NAME?=github.com/tigera/operator
 LOCAL_USER_ID?=$(shell id -u $$USER)
 GO_BUILD_VER?=v0.90
-CALICO_BUILD?=calico/go-build:$(GO_BUILD_VER)-$(ARCH)
+CALICO_BUILD?=calico/go-build:$(GO_BUILD_VER)-$(BUILDARCH)
```

Comment:
Use host native `calico/go-build` image for cross compiling all supported architectures.